packages/beacon-node/src/db/single/backfillRange.ts

```diff
@@ -0,0 +1,47 @@
+import {ContainerType, Type, ValueOf} from "@chainsafe/ssz";
+import {ChainForkConfig} from "@lodestar/config";
+import {Db, DbReqOpts, encodeKey as _encodeKey} from "@lodestar/db";
+import {ssz} from "@lodestar/types";
+import {Bucket, getBucketNameByValue} from "../buckets.js";
+
+export const backfillRangeSSZ = new ContainerType(
+  {
+    beginningEpoch: ssz.UintNum64,
```

Comment:
maybe use `Epoch` from `@lodestar/types/lib/sszTypes.js`

Reply:
Yes!  Good idea!  Use the same for `endingEpoch` as well.
```suggestion
    beginningEpoch: ssz.Epoch,
```